hmmlearn/base.py

```diff
@@ -405,6 +406,13 @@ def sample(self, n=1, random_state=None):
                 currstate, random_state=random_state))
 
         return np.array(obs), np.array(hidden_states, dtype=int)
+    
+    def _is_identifiable(self, means, n_components):
```

Comment:
I think this belongs to the `_init` method of each particular subclass, e.g. `GaussianHMM`.

Reply:
I think the floating-point equality problem is a real one here.  (Even though there can be cases of exact equality as demonstrated above, in my experience approximate equality is much more common.)